vlasovsolver/cpu_acc_intersections.cpp

```diff
@@ -29,7 +29,7 @@
 //#include <boost/unordered_map.hpp>
 //#include <boost/unordered_set.hpp>
 
-#include <Eigen/Geometry>
+#include <Geometry> //Eigen
```

Comment:
These Eigen include changes break compilation for me on my fridge. Are you sure the makefile you're using shouldn't just have its include paths changed?

Reply:
I had to do that when installing on HazelHen. I had the impression that there was one layer of directories too much or which had disappeared int he version of Eigen I took now. Maybe it was my mistake.